PushButtton/CS/README.md

```diff
@@ -0,0 +1,178 @@
+---
+page_type: sample
+urlFragment: push-button
+languages:
+  - csharp
+products:
+  - Windows 10
+  - Windows IoT 
+  - Windows 10 IoT Enterprise
+description: Make an LED blink with the push of a button with Windows 10 IoT Enterprise.
+---
+
+# Push button
+
+In this sample, we connect a push button to your UP Board and use it to control an LED. We use GPIO interrupts to detect when the button is pressed and toggle the LED in response.
+
+![Push Button Image](../../Resources/PushButtonSample.png)
+
+Also, be aware that the GPIO APIs are only available on boards, so this sample cannot run on your desktop.
+
+## Components
+
+You will need the following components:
+
+* [EG1311-ND Tactile Button](http://www.digikey.com/product-detail/en/320.02E11.08BLK/EG1311-ND/101397)
+
+* [Red LED](http://www.digikey.com/product-detail/en/C5SMF-RJS-CT0W0BB1/C5SMF-RJS-CT0W0BB1-ND/2341832)
+
+* [330 &#x2126; resistor](http://www.digikey.com/product-detail/en/CFR-25JB-52-330R/330QBK-ND/1636)
+
+* Breadboard and several male-to-male for the UP Board.
+
+### Connect the circuit to your device
+
+Let's start by wiring up the components on a breadboard. Visit the corresponding **UP Board** sections below depending on your device.

```

Comment:
Only one device is listed in this readme, so the sentence "Visit the corresponding **UP Board** sections below depending on your device." is unnecessary.

Reply:
Made the changes